### PR TITLE
Spell

### PR DESCRIPTION
**Pull request by sithumranabahu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,8 +79,8 @@ Contents
    * [Custom features](#custom-features)
    * [diff-highlight and diff-so-fancy emulation](#diff-highlight-and-diff-so-fancy-emulation)
    * [--color-moved support](#--color-moved-support)
-   * [Navigation keybindings for large diffs](#navigation-keybindings-for-large-diffs)
-   * [24 bit color (truecolor)](#24-bit-color-truecolor)
+   * [Navigation key bindings for large diffs](#navigation-key bindings-for-large-diffs)
```
Comment (Kr1ss-XD) [inline README.md:82]: Caution; this breaks the link.

**Comment by ryuta69** [comment]
Hi @Kr1ss-XD,

This user is definitely #shitoberfest user sending spam PR to OSS.

### [about shitoberfest]
https://twitter.com/joelnet/status/1311773094856454144?s=20
https://hacktoberfest.digitalocean.com/hacktoberfest-update
https://twitter.com/shitoberfest/status/1311646233128181760?s=20

I think it's good to refuse dealing with him, and close this PR.

**Comment by Kr1ss-XD** [comment]
Well thx for the heads up. :+1:

**Comment by dandavison** [comment]
Hey everyone, and thanks @Kr1ss-XD and @ryuta69. Yeah, I realized this morning that this PR must be related to the hacktoberfest fiasco.

@sithumranabahu this isn't a helpful contribution to the project, but if you'd like to help out properly just let us know.

**Pull request closed without merging**